src/test/java/com/jsoniter/suite/AllTestCases.java

```diff
@@ -56,6 +56,7 @@
         TestCollection.class,
         TestList.class,
         TestAnnotationJsonObject.class,
+        TestOmitValue.class,
```

Comment:
As this list seems to be in random order, should we assume it is based on chronology and add the new one last of all?